src/Notifications/data/api.test.js

```diff
@@ -62,14 +62,11 @@ describe('Notifications API', () => {
   });
 
   it('Successfully get notifications.', async () => {
-    axiosMock.onGet(notificationsApiUrl).reply(
-      200,
-      (Factory.buildList('notification', 2, null, { createdDate: new Date().toISOString() })),
-    );
+    axiosMock.onGet(notificationsApiUrl).reply(200, (Factory.build('notificationsList')));
 
-    const { notifications } = await getNotifications('discussions', 1, 10);
+    const notifications = await getNotificationsList('discussion', 1);
 
-    expect(notifications).toHaveLength(2);
+    expect(Object.keys(notifications.results)).toHaveLength(2);
```

Comment:
results is an array. notifications.results?.length

Reply:
results is an array instead of object in API response and also in factory. 
use 
expect(notifications.results).toHaveLength(2);